app/helpers/admin/tabbed_nav_helper.rb

```diff
@@ -1,4 +1,77 @@
 module Admin::TabbedNavHelper
+  include Admin::EditionsHelper
+
+  def secondary_navigation_tabs_items(model, current_path)
```

Comment:
I did think about doing something similar to this then alias_methoding the descendants but it seemed a bit overkill 

https://stackoverflow.com/questions/2393697/look-up-all-descendants-of-a-class-in-ruby